methods.py

```diff
@@ -134,6 +134,20 @@ def parse_cg_file(fname, uniforms, sizes, conditionals):
     fs.close()
 
 
+def get_cmdline_bool(option, default):
+    """We use `ARGUMENTS.get()` to check if options were manually overridden on the command line,
+    and SCons' _text2bool helper to convert them to booleans, otherwise they're handled as strings.
+    """
+    from SCons.Script import ARGUMENTS
+    from SCons.Variables.BoolVariable import _text2bool
+
+    cmdline_val = ARGUMENTS.get(option)
+    if cmdline_val != None:
```

Comment:
`None` is a singleton, hence it's more pythonic to do
```python
if cmdline_val is not None:
```

Reply:
Thanks, fixed.